Sources/SwiftSyntaxBuilder/ArrayElementConvenienceInitializers.swift

```diff
@@ -0,0 +1,19 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftSyntax
+
+extension ArrayElement: ExpressibleAsArrayExpr {
+  public func createArrayExpr() -> ArrayExpr {
+    ArrayExpr(elements: ArrayElementList([self]))
```

Comment:
I’m not entirely sure whether I am too fond of this. I think we should strive to make `ArrayElement` as transparent as possible (i.e. make sure that we can implicitly convert `Expr` to `ArrayElement` but I would prefer to keep an explicit wrap to `ArrayExpr` because it is modifying the semantic meaning.